docs/README.md

```diff
@@ -5,7 +5,9 @@ description: The home page for developer documentation for IPFS, the InterPlanet
 
 # Welcome to the IPFS docs
 
-The InterPlanetary File System (IPFS) is a decentralized file system for building the next generation of the internet. [Filecoin](https://filecoin.io) and many popular Web3 projects are built on IPFS. Some call it the hard drive for blockchain and Web3, though its power extends much further.
+The InterPlanetary File System (IPFS) is a set of composable, peer-to-peer protocols for addressing, routing, and transferring [content-addressed](concepts/glossary.md#content-addressing) data in a decentralized file system. Many popular Web3 projects are built on IPFS, check out the [ecosystem directory](https://ecosystem.ipfs.tech) for a list of some of these projects.
```

Comment:
@TMoMoreau nit RE phrasing of _Many popular Web3 projects are built on IPFS, check out the [ecosystem directory](https://ecosystem.ipfs.tech) for a list of some of these projects._

Currently, the phrasing here feels too casual. I think that removing  or adjusting the "... ,check out the ..." portion of the sentence would help. Ideas:


 _[Many popular Web3 projects are built on IPFS](https://ecosystem.ipfs.tech)._
 
although that sentence is not the best either because I feel like it just hangs. You could also write something like

_Many popular Web3 projects are built on IPFS - see the [ecosystem directory](https://ecosystem.ipfs.tech) for a list of some of these projects_

Reply:
This is partially personal preference, I do think you sentence still communicates the point

Reply:
Solid suggestion, I wasn't a fan of the way I wrote it either, just couldn't come up with anything better at the time.

I'll make this change.